spec/spec_helper.rb

```diff
@@ -2,6 +2,9 @@
 SimpleCov.start
 
 require 'byebug'
+require 'sidekiq/testing'
+require 'dotenv'
+Dotenv.load('.env.test')
```

Comment:
You may want to use `overload` rather than `load` here, as `load` won't replace any existing variables only add new ones, whereas `overload` will both add and replace.